dotcom-rendering/src/components/QuoteIcon.tsx

```diff
@@ -1,11 +1,11 @@
 import { css } from '@emotion/react';
 
 const quoteStyles = (colour: string) => css`
-	height: 1em;
-	width: 1.5em;
-	margin-right: 3px;
+	height: 0.75em;
+	width: auto;
 	vertical-align: baseline;
 	fill: ${colour};
```

Comment:
Should this colour be `palette('--pullquote-icon')`?

Reply:
We're passing different colours to this method, e.g.:

https://github.com/guardian/dotcom-rendering/blob/92fdc75750cb6aa3bc474d754901517e17ab6330/dotcom-rendering/src/components/PullQuoteBlockComponent.tsx#L191

https://github.com/guardian/dotcom-rendering/blob/92fdc75750cb6aa3bc474d754901517e17ab6330/dotcom-rendering/src/components/BlockquoteBlockComponent.tsx#L69

Reply:
Great, I’ll make a note that we should be using the `style` property then, [as per Emotion Best Practices](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles)!